README.md

```diff
@@ -30,4 +68,4 @@ The launched emulator will run at `localhost:8080` and will not have any persist
 To change the configuration see `./script/start-datastore.*` scripts.
 
 The datastore is cleaned up after each test.
-See test classes under `./gcd/src/test/java/...` and `TestDatastoreStorageFactory#clear`.
\ No newline at end of file
+See test classes under `./gcd/src/test/java/...` and `TestDatastoreStorageFactory#clear`.
```

Comment:
Please reference those via a package name. That will help the IDE to pick that up in case of package renaming.